src/components/rocket/rocketitem.js

```diff
@@ -0,0 +1,57 @@
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { reserveRockets, cancelResevations } from '../redux/rocket/rocketsSlice';
+import '../styles/rocketcards.css';
+
+function Rocketitem({
+  id,
+  image1,
+  summary,
+  title,
+  active,
+}) {
+  const dispatch = useDispatch();
+  const activeStatus = active;
+
+  const reserveRocket = (e, currId) => {
+    e.preventDefault();
+    dispatch(reserveRockets(currId));
+  };
+
+  const cancelRocket = (e, currId) => {
+    e.preventDefault();
+    dispatch(cancelResevations(currId));
+  };
+
+  return (
+    <li className="cards">
+      <div>
+        <img src={image1} alt="Rocket" width="200" />
+      </div>
+      <div className="contentSection">
+        <h2>{title}</h2>
+        <p className="description">
+          {activeStatus === true ? (
+            <span className="reservedTag">Reserved</span>
+          ) : null}
+          {summary}
+        </p>
+        <p>
+          { activeStatus === true
+            ? (<button className="cancelbtn" type="submit" onClick={(e) => cancelRocket(e, id)}>Cancel Resevation</button>)
+            : (<button type="submit" onClick={(e) => reserveRocket(e, id)}>Reserve Rocket</button>)}
```

Comment:
- [ ] The `activeStatus` for the buttons should be `Reserve Rocket` but there are two items that are reserved by default, please 🙏🏽 correct this issue to have all the rockets available for reservation at the beginning.

![image](https://github.com/geekelo/space-travelers-hub/assets/58642949/cc7f5778-6f07-4bd4-82b8-3d15a8af01aa)

Reply:
Thanks @franclobo

Reply:
- Well resolved @geekelo 👍🏽